lib/contract/init/init.go

```diff
@@ -0,0 +1,7 @@
+package init
+
+import "boscoin.io/sebak/lib/contract/native/execfunc"
+
+func init() {
+	execfunc.Init()
+}
```

Comment:
Is not good `exexfunc.init` ?

Reply:
good point i will change. 
It was influenced by the other project.